client/views/components/generic/time-ago-in-words/TimeAgoInWords.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {PureComponent } from 'react';
+import { string, object  } from 'prop-types';
+import { BaseEntityQuery } from 'utils/genny';
+import distanceInWords from 'date-fns/distance_in_words';
+
+class TimeAgoInWords extends PureComponent {
+    static propTypes = {
+        code: string,
+        style: object
+    }
+    renderFromNow = (code) => {
+      if (code) {
+        let createdAt = BaseEntityQuery.getBaseEntity(code);
+        if(createdAt && createdAt.created) { 
+          const date = createdAt.created;
+          return distanceInWords(date, Date.now());
+        }
+        else { 
```

Comment:
unnecessary `else` statements, can just put `return null` outside the `if` statement